src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -157,45 +157,23 @@
-                        "The tool collects fingerprints (essentially, genotype information from different parts of the genome) " +
-                        "at the finest level available in the data (readgroup for read-data files " +
-                        "and sample for variant-data files) and then optionally aggregates it by library, sample or file, to increase power and provide " +
-                        "results at the desired resolution. Output is in a \"Moltenized\" format, one row per comparison. The results are " +
-                        "emitted into a CrosscheckMetric metric file. " +
-                        "In this format the output will include the LOD score and also tumor-aware LOD score which can " +
-                        "help assess identity even in the presence of a severe loss of heterozygosity with high purity (which could cause it to " +
-                        "otherwise fail to notice that samples are from the same individual.) " +
-                        "A matrix output is also available to facilitate visual inspection of crosscheck results.\n " +
+                        "CrosscheckFingerprints rapidly checks the odds that all of the genetic data within a set of files come from the same individual. This is accomplished by selectively sampling from the input files, and determining whether the genotypes of the specified readgroups match to each other. \n " +
+                        "Output is generated in the form of a “molten” (one row per comparison) CrosscheckMetric file that includes the Logarithm of the Odds (LOD) score, as well as the tumor-aware LOD score. Tumor-aware LOD scores can be used to assess genotypic identity in the presence of a severe Loss of Heterozygosity (LOH) with high purity—this could otherwise lead to a failure of the tool to identify samples are from the same individual. Output is also available as a matrix, to facilitate visual inspection of crosscheck results. \n" +
+                        "Metric files can contain many rows of output. We therefore recommend following up CrosscheckFingerprints with a step using [ClusterCrosscheckMetrics (Picard)](https://gatk.broadinstitute.org/hc/en-us/articles/360045798972--Tool-Documentation-Index); this tool will cluster groups together that pass a designated LOD threshold, ensuring that groups within the cluster are related to each other. \n" +
+                        "There may be cases where several groups out of a collection of possible groups must be identified—for example, to link a BAM to its correct sample in a multi-sample VCF. In this case, it would not be necessary to cross-check the various samples in the VCF against each other, but only to check the identity of the BAM against the various samples in the VCF. For this application, the SECOND_INPUT argument is provided. With SECOND_INPUT, CrosscheckFingerprints does the following: \n" +
```

Comment:
```suggestion
                        "There may be cases where several groups out of a collection of possible groups must be identified---for example, to link a BAM to its correct sample in a multi-sample VCF. In this case, it would not be necessary to cross-check the various samples in the VCF against each other, but only to check the identity of the BAM against the various samples in the VCF. For this application, the SECOND_INPUT argument is provided. With SECOND_INPUT, CrosscheckFingerprints does the following: \n" +
```